app/policies/comment_policy.rb

```diff
@@ -0,0 +1,5 @@
+class CommentPolicy < ApplicationPolicy
+  def owner?
```

Comment:
owner nav slikts nosaukums, bet kā ir ar admin/moderator lietotājiem? Ah, tev vēl tādu nav.

Nu ok.

te piedāvājums, bet ne obligāti realizējams:
1) aliaso update, create un destroy actionus, tad kontrolierī varēsi actionos rakstīt vnk `authorize @comment`

2) dažreiz mikrooptimizācija ir noderīga. Skaties - es salīdzinu user.id ar record.user_id, tādā veidā es neveidoju pieprasījumu uz db, lai ielādētu user instanici, kuru actualli nemaz neizmantojam.
tavā gadījumā šis gan neko īsti nemaina, jo tev nav nekādu performance issues, bet dažreiz der padomāt par to un tādas lietas ir nepieciešamas. Bet protams smukāk izskatās `user == record.user`

```ruby
def owner?
  user.id == record.user_id
end

def update?
  owner?
end

def create?
  owner?
end

def destroy?
  owner?
end
```